### DISCUSSION TITLE
can yall give me galaxy brain???

### DISCUSSION THREAD

**Discussion by ghost** [drknzz/GitHub-Achievements#323] [Q&A]
its my dreammm

**Comment by yubrajpandeya** [comment]
You need to do all that is one of your answer should be upvoated♥️♥️✅✅✅😊

**Reply by ghost** [reply]
Ok thanks!

**Comment by davevad93** [comment]
Hi @CarelessZebra2134 ,

The `Galaxy Brain` badge can no longer be obtained via Community Discussions, but you can get it through other GitHub discussions like this one, for instance you can open a discussions section in one of your public repos and get a minimum of 2 answers marked as `answered` by another user. If you want to get the badge I think that would be the easiest way to earn it.
